src/data/cloud/tts.ts

```diff
@@ -60,7 +60,7 @@ export const getCloudTtsSupportedGenders = (
       genders.push([
         gender,
         gender === "male" || gender === "female"
-          ? localize(`ui.panel.config.cloud.account.tts.${gender}`)
```

Comment:
If this key didn't exist, TypeScript would throw an error.  Is the key wrong?

Reply:
No, but the config translations will not be loaded in the media browser, they are only loaded on the config panel

Reply:
Should they be removed from the translations now? Or are they used somewhere else still?

Reply:
Ohhh... if I remember correctly, this was the only spot so they can probably be removed.

Reply:
Yes, I will remove them 🙂